ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -328,13 +328,14 @@ private[kvutils] class TransactionCommitter(
     if (!transactionEntry.submitterInfo.hasDeduplicationDuration) {
       throw Err.InvalidSubmission("Deduplication duration is not set.")
     }
-    val commandDedupBuilder = DamlCommandDedupValue.newBuilder.setDeduplicatedUntil(
-      Conversions.buildTimestamp(
-        transactionEntry.submissionTime
-          .add(parseDuration(transactionEntry.submitterInfo.getDeduplicationDuration))
-          .add(config.timeModel.minSkew)
-      )
+    val deduplicateUntil = Conversions.buildTimestamp(
+      transactionEntry.submissionTime
+        .add(parseDuration(transactionEntry.submitterInfo.getDeduplicationDuration))
+        .add(config.timeModel.minSkew)
```

Comment:
Can you add a comment about why we're using the min skew here?

Reply:
This is taken from the design doc. I could add my understanding of it but I would prefer to leave it as is for now, and add a proper comment in the follow up PR which changes this logic